Sources/NIO/EventLoop.swift

```diff
@@ -222,6 +291,66 @@ extension EventLoop {
     public func close() throws {
         // Do nothing
     }
+
+    /// Schedule a repeated task to be executed by the `EventLoop` in a fixed rate between the start of tasks.
+    ///
+    /// - parameters:
+    ///     - initialDelay: The delay after which the task is initally executed.
+    ///     - period: The period between the start of tasks.
+    ///     - task: The closure that will be executed.
+    /// - return: `RepeatedTask`
+    @discardableResult
+    public func scheduleTaskAtFixedRate(initialDelay: TimeAmount, period: TimeAmount, _ task: @escaping () throws -> Void) -> RepeatedTask {
+        let futureTask: () throws -> EventLoopFuture<Void> = {
+            try task()
+            return self.newSucceededFuture(result: ())
+        }
+        return scheduleTaskAtFixedRate(initialDelay: initialDelay, period: period, futureTask)
+    }
+
+    /// Schedule a repeated task to be executed by the `EventLoop` in a fixed rate between the start of tasks.
+    ///
+    /// - parameters:
+    ///     - initialDelay: The delay after which the task is initally executed.
+    ///     - period: The period between the start of tasks.
+    ///     - task: The closure that will be executed.
+    /// - return: `RepeatedTask`
+    @discardableResult
+    public func scheduleTaskAtFixedRate(initialDelay: TimeAmount, period: TimeAmount, _ task: @escaping () throws -> EventLoopFuture<Void>) -> RepeatedTask {
```

Comment:
- we need a test for this
- this shouldn't be `throws` (as it's returning an `EventLoopFuture`)